README.md

```diff
@@ -1,2 +1,81 @@
 # Confluence
 This is Confluence of Conference.
+
+# Setup Guide
+
+### Creating a Virtual Environment
+Virtual Environment allows us to have requirements installed specifically to our project instead of complete system. To create a virtual environment goto any directory where you will store your code and run the following command on terminal (linux) or Windows PowerShell (Windows)(For windows, run Windows Powershell as Administrator before running below commands)
+```
+python -m venv venvconf
+```
+#### Activating the Virtual Environment
+Run the following command to activate your virtual environment:
+##### Windows
+```
+./venvconf/Scripts/activate
+```
+##### Linux
+```
+source venvconf/bin/activate
+```
+**Note:** To deactivate the Virtual Environment, type deactivate in the shell to deactivate.
+#### Clone your forked repo
+```
+git clone https://github.com/your_username/Confluence
```

Comment:
Mention `<github_username>` instead of `your_username`

Also, add a note that people have to replace this with their github username